client/blocks/nps-survey/index.jsx

```diff
@@ -71,45 +75,51 @@ class NpsSurvey extends Component {
 		const shouldDisableControls = this.state.isSubmitting || this.state.isSubmitted;
 
 		return (
-			<div className={ className }>
+			<Card className={ className }>
 				<div className="nps-survey__question-screen">
-					<div>How likely is it that you would recommend WordPress.com to your friends, family, or colleagues?</div>
-					<div>
+					<div className="nps-survey__question">
+						{ translate( 'How likely are you to recommend WordPress.com to your friends, family, or colleagues?' ) }
+					</div>
+					<div className="nps-survey__recommendation-select-wrapper">
 						<RecommendationSelect
 							value={ this.state.recommendationValue }
 							disabled={ shouldDisableControls }
 							onChange={ this.handleRecommendationSelectChange }
 						/>
 					</div>
-					<div>
+					<div className="nps-survey__buttons">
 						<Button primary
 							className="nps-survey__finish-button"
 							disabled={ shouldDisableControls }
 							onClick={ this.handleFinishClick }
 						>
-							Finish
+							{ translate( 'Finish', { context: 'verb' } ) }
 						</Button>
 						<Button borderless
+							className="nps-survey__not-answer-button"
 							disabled={ shouldDisableControls }
 							onClick={ this.handleDismissClick }
 						>
-							I'd rather not answer
+							{ translate( 'I\'d rather not answer' ) }
 						</Button>
 					</div>
 				</div>
 				<div className="nps-survey__thank-you-screen">
-					Thanks for providing your feedback!
-					<div>
+					<div className="nps-survey__thank-you">
+						{ translate( 'Thanks for your feedback!' ) }
```

Comment:
Hi! I've found a possible matching string that has already been translated **27** times:
`translate( 'Thanks for your feedback.', { context: 'calypso stats survey'} )` *ES Score: 14*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).